manager/migrations/0001_initial.py

```diff
@@ -0,0 +1,90 @@
+                ('final_date', models.DateField()),
+                ('status', models.BooleanField()),
+            ],
+        ),
+        migrations.CreateModel(
+            name='Stand',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('size', models.IntegerField(choices=[(1, 'Small'), (2, 'Medium'), (3, 'Large')])),
+            ],
+        ),
+        migrations.CreateModel(
+            name='StandRequest',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('initial_date', models.DateField()),
+                ('final_date', models.DateField()),
+                ('entity', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to=settings.AUTH_USER_MODEL)),
+                ('event', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='manager.event')),
+                ('stand', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='manager.stand')),
+            ],
+        ),
+        migrations.CreateModel(
+            name='StandContract',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('file', models.FileField(upload_to='')),
+                ('entity', models.ForeignKey(null=True, on_delete=django.db.models.deletion.SET_NULL, to=settings.AUTH_USER_MODEL)),
+                ('stand', models.ForeignKey(null=True, on_delete=django.db.models.deletion.SET_NULL, to='manager.stand')),
+            ],
+        ),
+        migrations.CreateModel(
+            name='ServiceRequest',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('entity', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to=settings.AUTH_USER_MODEL)),
+                ('event', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='manager.event')),
+                ('service', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='manager.additionalservice')),
+                ('stand', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='manager.stand')),
+            ],
+        ),
+        migrations.CreateModel(
+            name='EventContract',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('file', models.FileField(upload_to='')),
+                ('entity', models.ForeignKey(null=True, on_delete=django.db.models.deletion.SET_NULL, to=settings.AUTH_USER_MODEL)),
+                ('event', models.ForeignKey(null=True, on_delete=django.db.models.deletion.SET_NULL, to='manager.event')),
+            ],
+        ),
+    ]
```

Comment:
I think it's better not to commit/push the migration files, there should be plenty of changes and they will be annoying in local development.

Reply:
Alright Alex, thanks for the heads up.